addons/website_sale/models/product_template.py

```diff
@@ -58,6 +67,15 @@ class ProductTemplate(models.Model):
         digits='Product Price',
         help="The amount will be displayed strikethroughed on the eCommerce product page")
 
+    @api.depends('ecommerce_description', 'description_sale')
+    def _compute_product_page_description(self):
+        for template in self:
+            template.product_page_description = template.ecommerce_description or (template.description_sale and f'<p class="text-muted my-2">{template.description_sale}</p>') or False
```

Comment:
```suggestion
            template.product_page_description = template.ecommerce_description or (template.description_sale and Markup('<p class="text-muted my-2"></p>') % template.description_sale) or False
```
`description-sale` is a text field, you want to escape it